### DISCUSSION TITLE
play 하면 에러가 납니다. Typescript sources is not compiled yet.

### DISCUSSION THREAD

**Discussion by hongtuff** [naverz/zepeto-studio-kor#971] [5. Q&A]
타입스크립트만 생성 하면 에러가 나면서 
play 누르면 Typescript sources is not compiled yet. 라고 뜨면서 안됩니다.
재설치도 수십번 하고 
다른글에서 zepeto.script 버전도 preview로 업데이트 해봤는데도 안되네요
![image](https://user-images.githubusercontent.com/56460638/177460379-61e442b1-5db5-4788-b6fb-9aec3d50179a.png)

어떻게 해야 할까요??

**Comment by YSHFM** [comment]
ZEPETO>Script>Rebuild All하셔도 동일한가요?

**Comment by hongtuff** [comment]
네 똑갔습니다.
달리지는게 없네요
-----Original Message-----

**Reply by YSHFM** [reply]
새로생성한 저 controll파일이 컴파일 에러나는 거라면,,,샘플스크립트도 컴파일이 안되어야 정상으로 보이는데 샘플 스크립트도 마찬가지로 컴파일이 안되시는 건가여....?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요 에디터 우측 하단의 파란색 TS 아이콘에 불이 꺼져 있지는 않는지 확인 부탁 드립니다.
타입스크립트 서버가 꺼지면 위와 같은 상황이 발생될 수 있으며
월드를 최신 버전으로 업데이트 하시고, **zepeto.script 패키지도 1.3.4 preview 버전으로 적용**해 보시기를 권장 드립니다.

[적용 방법]
Window → Package Manager 버튼을 눌러서 Package Manager를 오픈 → [ZEPETO.Script]를 누르고 오른쪽 하단의 Update 버튼을 누릅니다.
1.3.4 preview 버전 업 후 컴파일러 재실행을 위해 유니티를 재시작 해 주세요.

<img width="797" alt="174784909-cdeea83d-0881-4f56-a6cf-7a58a9a4415c" src="https://user-images.githubusercontent.com/100118650/177770302-c9370b9e-fbf5-4513-87c7-42d434572a99.png">
